include/lightweight_filtering/OutlierDetection.hpp

```diff
@@ -44,7 +44,7 @@ class OutlierDetectionBase{
     outlier_ = d_ > mahalanobisTh_;
     if(outlier_){
       outlierCount_++;
-      VLOG(2) << "Detected outlier - mahalanobis (d / threshold):"  << d_
+      VLOG(4) << "Detected outlier - mahalanobis (d / threshold):"  << d_
```

Comment:
```suggestion
      VLOG(2) << "Detected outlier - mahalanobis (d / threshold):"  << d_
```
i would keep this at a lower level to avoid hiding problem e.g. with the calibration

Reply:
done